### DISCUSSION TITLE
Is there already STM32G0B1CBT6 Support

### DISCUSSION THREAD

**Discussion by alextrical** [stm32duino/Arduino_Core_STM32#1773] [Q&A] [answered]
Good day.

I'm currently trying to use the STM32G0B1CBT6 with your core, but slightly confused as to whether the definition is already there and I'm just overlooking it, or if I need to define it myself.

I see there is reference to "ARDUINO_GENERIC_G0B1CBTX" in "hardware\stm32\2.3.0\variants\STM32G0xx\G0B1C(B-C-E)(T-U)_G0C1C(C-E)(T-U)\variant_generic.cpp" however I cannot seem to locate the matching entry in the Arduino 1.8.19 IDE. Please advise if I'm missing something obvious (I cant find the keywords for any similar issues), as I would like to use this as the core chip for the Open source project, PolyFormer https://github.com/Reiten966/Polyformer
![image](https://user-images.githubusercontent.com/35117191/180604964-6147d4d4-e440-488f-a06f-551be857a001.png)

Many thanks for any advice
Alextrical

**Comment by alextrical** [comment]
The solution looks to be issues to be fixable by copying the contents (excluding header) of ...\STMicroelectronics\hardware\stm32\2.3.0\variants\STM32G0xx\G0B1C(B-C-E)(T-U)_G0C1C(C-E)(T-U)\boards_entry.txt to the end of ...\STMicroelectronics\hardware\stm32\2.3.0\boards.txt. And adding the ldscript.ld (to the folder containing boards_entry.txt) generated by following the guide.
 
[ldscript.zip](https://github.com/stm32duino/Arduino_Core_STM32/files/9174292/ldscript.zip)

**Reply by fpistm** [reply, marked as answer]
Hi @alextrical 
yes you have to declare it, see this wiki page:
https://github.com/stm32duino/wiki/wiki/Add-a-new-variant-%28board%29
